cockatrice/src/tab_deck_editor.cpp

```diff
@@ -764,6 +771,19 @@ void TabDeckEditor::actPrintDeck()
     dlg->exec();
 }
 
+void TabDeckEditor::actExportDeckDecklist()
+{
+    DeckLoader *const deck = deckModel->getDeckList();
+    if(deck != NULL){
```

Comment:
Use `nullptr`, not the `NULL` macro please.

Reply:
I'm also fine if he just says `if (deck) {`

Reply:
Ok I like that better personally. I'll switch it back :)